File: C++/problem13.c
#include <iostream>
#include <fstream>
#include <iomanip>
using namespace std;

int main()
{
  ifstream is("Problem13.txt");
  double numbers[100];
  double sum = 0;
  for (int i = 0; i < 100; i++)
    is >> numbers[i];
  

  for (int i = 0; i < 100; i++)
    sum += numbers[i];
  
  cout << setprecision(100) << sum << endl;
}

File: problem13.py
def read_file() -> list[int]:
    with open("problem13.txt", "r") as fp:
        return [int(line) for line in fp.readlines()]


def solve(numbers: list[int]) -> int:
    sigma: int = sum(numbers)
    return int(str(sigma)[:10])


if __name__ == "__main__":
    numbers: list[int] = read_file()
    print(solve(numbers))
